pkg/xray/resource_xray_watch_test.go

```diff
@@ -99,18 +99,18 @@ func TestAccWatch_singleRepository(t *testing.T) {
 	tempStruct["watch_name"] = fmt.Sprintf("xray-watch-%d", randomInt())
 	tempStruct["policy_name_0"] = fmt.Sprintf("xray-policy-%d", randomInt())
 	tempStruct["watch_type"] = "repository"
-	repo0 := "libs-release-local"
-	repo1 := "libs-release-local-1"
+	tempStruct["repo0"] = fmt.Sprintf("libs-release-local-0-%d", randomInt())
```

Comment:
Why do we need to use a struct?

Reply:
it's not a struct, it's a map of strings. Just the name, which I borrowed in the Artifactory provider.

Reply:
Ok, why are we using a map of strings? Why is this necessary?

Reply:
I'm using the template to construct HCL. Template uses the map to fill in values into the template

Reply:
Ok then please rename it with more descriptive name. `tempStruct` is just wrong. It isn't temporary, or a struct. It tells me nothing about what it is for.

Reply:
renamed to testData